apps/smart-contracts/core/test/PrePOMarket.test.ts

```diff
@@ -68,6 +69,8 @@ describe('=> prePOMarket', () => {
       factory: prePOMarketFactory,
       tokenNameSuffix: TEST_NAME_SUFFIX,
       tokenSymbolSuffix: TEST_SYMBOL_SUFFIX,
+      longTokenSalt: formatBytes32String('DEFAULT_SALT'),
+      shortTokenSalt: formatBytes32String('DEFAULT_SALT'),
```

Comment:
nit: should use different values to be more realistic, same in the other file
```suggestion
      longTokenSalt: formatBytes32String('LONG_SALT'),
      shortTokenSalt: formatBytes32String('SHORT_SALT'),
```